static/css/tools/rich_search.scss

```diff
@@ -0,0 +1,591 @@
+/**
```

Comment:
is this a copy of the place css? please only pull in the css you need, and add more as you go along.. it's harder to extract after the fact and its ok to have a UI that's not-so-great while we iterate..